app/controllers/application_controller.rb

```diff
@@ -137,4 +137,9 @@ def meta_tag_description
   def requested_organisation_path
     organisation_path(Organisation.find(current_user.pending_organisation_id))
   end
+
+  def send_email_to_superadmin_about_request_for_admin_of org
```

Comment:
I really love how you realize that the code is repeated and for sure we need to refactor the code. But I think that by moving this functionality to `ApplicationController` you are making this method available to all controllers, and we only want to use it in `RegistrationsController` and `UsersController`.  One alternative solution and to make this method only available to the controllers we want, it will be to create a new service holding this functionality. This service then it will be included in both controllers and it will make us aware that they are using this service to perform this functionality. What do you think @sigu ?

Reply:
Thank you, makes sense, lemme make changes.